### PR TITLE
L2 bridges alerts

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: l2-bridge-arbitrum/src/constants.ts [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    type: FindingType.Info,
+  },
+  {
+    address: GOV_BRIDGE_ADDRESS,
+    event:
+      "event MinimumDelayUpdate(uint256 oldMinimumDelay, uint256 newMinimumDelay)",
+    alertId: "GOV-BRIDGE-MIN-DELAY-UPDATED",
+    name: "Arbitrum Gov Bridge: Min Delay Updated",
+    description: (args: any) =>
+      `Ethereum Governance Executor was updated from ` +
```
Comment (madlabman) [inline l2-bridge-arbitrum/src/constants.ts:79]: Should be minimum delay

File: l2-bridge-arbitrum/src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    type: FindingType.Info,
+  },
+  {
+    address: GOV_BRIDGE_ADDRESS,
+    event:
+      "event MaximumDelayUpdate(uint256 oldMaximumDelay, uint256 newMaximumDelay)",
+    alertId: "GOV-BRIDGE-MAX-DELAY-UPDATED",
+    name: "Arbitrum Gov Bridge: Max Delay Updated",
+    description: (args: any) =>
+      `Ethereum Governance Executor was updated from ` +
```
Comment (madlabman) [inline l2-bridge-arbitrum/src/constants.ts:91]: Same as above.

File: l2-bridge-optimism/src/agent-proxy-watcher.ts [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+export async function handleTransaction(txEvent: TransactionEvent) {
+  const findings: Finding[] = [];
+
+  handleGovBridgeEvents(txEvent, findings);
+
+  return findings;
+}
+
+function handleGovBridgeEvents(txEvent: TransactionEvent, findings: Finding[]) {
```
Comment (madlabman) [inline l2-bridge-optimism/src/agent-proxy-watcher.ts:50]:
```suggestion
function handleProxyAdminEvents(txEvent: TransactionEvent, findings: Finding[]) {
```

File: l2-bridge-arbitrum/src/agent-proxy-watcher.ts [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+export async function handleTransaction(txEvent: TransactionEvent) {
+  const findings: Finding[] = [];
+
+  handleGovBridgeEvents(txEvent, findings);
+
+  return findings;
+}
+
+function handleGovBridgeEvents(txEvent: TransactionEvent, findings: Finding[]) {
```
Comment (madlabman) [inline l2-bridge-arbitrum/src/agent-proxy-watcher.ts:50]:
```suggestion
function handleProxyAdminEvents(txEvent: TransactionEvent, findings: Finding[]) {
```

File: l2-bridge-ethereum/src/agent.ts [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+      alertId: "LIDO-AGENT-LAUNCHED",
+      severity: FindingSeverity.Info,
+      type: FindingType.Info,
+      metadata,
+    })
+  );
+  console.log("Bot initialization is done!");
+};
+
+const handleBlock: HandleBlock = async (
```
Comment (madlabman) [inline l2-bridge-ethereum/src/agent.ts:100]: What about an updated implementation from `main` HEAD?

File: l2-bridge-arbitrum/src/constants.ts (hunk 1 above)
Comment (dgusakov) [reply l2-bridge-arbitrum/src/constants.ts]: Fixed

File: l2-bridge-arbitrum/src/constants.ts (hunk 2 above)
Comment (dgusakov) [reply l2-bridge-arbitrum/src/constants.ts]: Fixed

File: l2-bridge-optimism/src/agent-proxy-watcher.ts (hunk 3 above)
Comment (dgusakov) [reply l2-bridge-optimism/src/agent-proxy-watcher.ts]: Fixed

File: l2-bridge-arbitrum/src/agent-proxy-watcher.ts (hunk 4 above)
Comment (dgusakov) [reply l2-bridge-arbitrum/src/agent-proxy-watcher.ts]: Fixed

File: l2-bridge-ethereum/src/agent.ts (hunk 5 above)
Comment (dgusakov) [reply l2-bridge-ethereum/src/agent.ts:100]: Updated

**Review by madlabman — APPROVED** [review]

File: l2-bridge-ethereum/src/agent.ts [hunk 6]
```diff
@@ -25,7 +26,10 @@ interface SubAgent {
   initialize?: (blockNumber: number) => Promise<Metadata>;
 }
 
-const subAgents: SubAgent[] = [agentProxy];
+const subAgents: SubAgent[] = [
+  //agentProxy,
```
Comment (madlabman) [inline l2-bridge-ethereum/src/agent.ts:30]: dev staff

Comment (dgusakov) [reply l2-bridge-ethereum/src/agent.ts]: fixed

**Review by madlabman — APPROVED** [review]

**Pull request merged**